src/ocm/components/clusterConfiguration/review/ReviewOperatorsTable.tsx

```diff
@@ -0,0 +1,67 @@
+import { Table, TableVariant, TableBody } from '@patternfly/react-table';
+import React from 'react';
+import {
+  Cluster,
+  hasEnabledOperators,
+  isSNO,
+  operatorLabels,
+  OPERATOR_NAME_CNV,
+  OPERATOR_NAME_LVM,
+  OPERATOR_NAME_ODF,
+} from '../../../../common';
+import { useTranslation } from '../../../../common/hooks/use-translation-wrapper';
+
+export const ReviewOperatorsTable = ({ cluster }: { cluster: Cluster }) => {
+  const { t } = useTranslation();
+  const operatorNames = operatorLabels(t);
+
+  const rows = React.useMemo(() => {
+    return [
+      {
+        cells: [
+          operatorNames[OPERATOR_NAME_CNV],
+          {
+            title: hasEnabledOperators(cluster.monitoredOperators, OPERATOR_NAME_CNV)
+              ? 'Enabled'
+              : 'Disabled',
+            props: { 'data-testId': 'openshift-virtualization' },
```

Comment:
@jgyselov @celdrake this should be 'data-testid' all in lowercase
It is all over the place, and produces the following error in the console:
![image](https://user-images.githubusercontent.com/77008341/209287749-ec3e528d-d8e2-474b-9e84-99fd327a2279.png)

Reply:
Well spotted, added a PR to fix it:
https://github.com/openshift-assisted/assisted-ui-lib/pull/1832